docs/vNext/Fabric_Configuration.md

```diff
@@ -39,23 +39,23 @@ To install the Fabric SDK dependencies of the adapter, configure the binding com
 
 You can set the above keys either from the command line:
 ```console
-user@ubuntu:~/caliper-benchmarks$ npx caliper bind --caliper-bind-sut fabric --caliper-bind-sdk 1.4.0
+user@ubuntu:~/caliper-benchmarks$ npx caliper bind --caliper-bind-sut fabric --caliper-bind-sdk 1.4.11
 ```
 or from environment variables:
 ```console
 user@ubuntu:~/caliper-benchmarks$ export CALIPER_BIND_SUT=fabric
-user@ubuntu:~/caliper-benchmarks$ export CALIPER_BIND_SDK=1.4.0
+user@ubuntu:~/caliper-benchmarks$ export CALIPER_BIND_SDK=1.4.11
 user@ubuntu:~/caliper-benchmarks$ npx caliper bind
 ```
 or from various [other sources](./Runtime_Configuration.md).
 
 ### Binding with Fabric SDK 2.0.x
```

Comment:
should we change the title here to match the 2.2 used below?

Reply:
we also list the explicit available bindings in the `vNext/installing-caliper/#the-bind-command` page .. could you add the new bindings there too?

Reply:
done :-)